### DISCUSSION TITLE
Build Error: Node Sass does not yet support your current environment: OS X Unsupported architecture (arm64) with Node.js 18.x

### DISCUSSION THREAD

**Discussion by pavinduLakshan** [wso2/oxygen-ui#208] [Q&A]
**Describe the issue:**

Has anyone got the following error when building oxygen-ui on Mac M1?

```
Error: Node Sass does not yet support your current environment: OS X Unsupported architecture (arm64) with Node.js 18.x
For more information on which environments are supported please see:
       https://github.com/sass/node-sass/releases/tag/v8.0.0
```

Seems the arm architecture is not supported in node-sass v8 [1], but I was able to build the repo a few months ago with this same version afair.

[1] https://github.com/sass/node-sass/releases/tag/v8.0.0

**How to reproduce:**

1. Checkout main branch.
2. Run `pnpm install` and `pnpm build`.

**Expected behavior:**

Should be able to build without an issue.

**Environment information** (_Please complete the following information; remove  any unnecessary fields_) **:**
 - Affected Package: [e.g., @oxygen-ui/react, @oxygen-ui/primitives]
 - Library Version: [e.g., 1.10.0, 2.0.0-alpha]
 - OS: Mac OS 14 Sonoma
 - Browser: N/A
 
---

### Optional Fields

**Related issues:**

**Suggested labels:**

**Comment by jeradrutnam** [comment]
Closing this discussion. This is no longer relevant, as we have a newer version of oxygen-ui
